7.4/cli/Dockerfile

```
@@ -43,6 +47,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     docker-php-ext-install sockets && \
     pecl install imagick && \
     docker-php-ext-enable imagick && \
+    pecl install ssh2-1.2 && \
+    docker-php-ext-enable ssh2 && \
     apt-get remove -y libfreetype6-dev libpng-dev libmagickwand-dev libjpeg62-turbo-dev ghc && \
```

```suggestion
    apt-get remove -y libfreetype6-dev libpng-dev libmagickwand-dev libjpeg62-turbo-dev ghc libssh2-1-dev && \
```

Тебе исходники после сборки уже не нужны. Аналогично ниже